src/components/MostPopular/ProductItem/ProductItem.js

```diff
@@ -1,34 +1,92 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
+import { NavLink } from 'react-router-dom';
+import Modal from '../../Modal';
+import ProductImage from '../../ProductImage';
+import ProductOrder from '../../ProductOrder';
+import Heading from '../../Heading';
+// import { defaultImages } from '../../../constants';
+import { productOrderParameters } from '../../../config/ProductOrderMockups';
 
 import './ProductItem.scss';
 
 const CN = 'product';
 
-const ProductItem = ({
-  product, wishlist, addToWishlist, removeFromWishlist
-}) => {
+const ProductItem = (props) => {
   const [isHovered, setHovered] = useState(false);
-
+  const [isShowedModal, setShowModal] = useState(false);
+  const {
+    item, wishlist, addToWishlist, removeFromWishlist, showMessage
+  } = props;
   const {
-    id,
-    image,
-    label,
-    title,
-    price,
-    sizes
-  } = product;
+    _id, images, label, title, price, sizes
+  } = item;
+
+  const notificationTypeEnum = {
+    removedFromWishlistNotif: {
+      title: 'info',
+      message: 'The item removed from the wishlist',
+      type: 'info'
+    },
+    addedItemNotif: {
+      title: 'success',
+      message: 'The item added to the wishlist',
+      type: 'success'
+    }
+  };
+
+  const audio = new Audio('src/assets/sounds/notification-sound.mp3');
+
+  const playAudio = () => {
+    audio.play();
+  };
```

Comment:
why do you use Audio here ? It should be in Notification Component

Reply:
This change has been added by Dima Hnatkivsky, I only pulled it from the development when I was updating my branch. I will consult with him and we will make proper changes on his branch.